compiler/quilt/tools/util.py

```diff
@@ -187,24 +193,31 @@ def to_identifier(string):
     >>> to_identifier('9foo') -> 'n9foo'
 
     :param string: string to convert
+    :param permit_keyword: Permit python keywords like "import" and "for"
     :returns: `string`, converted to python identifier if needed
     :rtype: string
     """
-    # Not really useful to expose as a CONSTANT, and python will compile and cache
-    string = re.sub(r'[^0-9a-zA-Z_]', '_', string)
+    # Not really useful to expose as a constant, and python will compile and cache
```

Comment:
Is this needed for export? If not, could you put this in a separate PR?

Reply:
Already done -- that's in #537